modules/simple_forms_api/app/models/simple_forms_api/vba_26_4555.rb

```diff
@@ -45,6 +45,23 @@ def zip_code_is_us_based
       @data.dig('veteran', 'address', 'country') == 'USA'
     end
 
+    def desired_stamps
+      desired_stamps = []
+      unless data['previous_sah_application']['has_previous_sah_application']
+        desired_stamps.append({ coords: [73, 390], text: 'X',
+                                page: 0 })
+      end
+      unless data['previous_hi_application']['has_previous_hi_application']
+        desired_stamps.append({ coords: [73, 355], text: 'X',
+                                page: 0 })
+      end
+      unless data['living_situation']['is_in_care_facility']
+        desired_stamps.append({ coords: [73, 320], text: 'X',
+                                page: 0 })
+      end
+      desired_stamps
```

Comment:
This could be streamlined with a `.tap` method call like so:

```suggestion
     return [] unless form.data

      [].tap do |stamps|
        stamps << [73, 390, 'X'] unless form.data.dig('previous_sah_application', 'has_previous_sah_application')
        stamps << [73, 355, 'X'] unless form.data.dig('previous_hi_application', 'has_previous_hi_application')
        stamps << [73, 320, 'X'] unless form.data.dig('living_situation', 'is_in_care_facility')
```